salt/modules/ansiblegate.py

```diff
@@ -147,6 +147,10 @@ def call(self, module, *args, **kwargs):
         :param kwargs: keywords to the module
         :return:
         '''
+        if six.PY3:
+            python_exec = 'python3'
+        else:
+            python_exec = 'python'
```

Comment:
Why not `sys.executable`?

Reply:
Because some distros which will eventually abandon Py2 will probably not have `python3` and it's `python` binary will actually be the Py3 one.

Reply:
Thanks for the review @s0undt3ch :+1:

You're right, I'm pushing the changes now.

Reply:
:+1: